docs/styles/globals.css

```diff
@@ -2,8 +2,8 @@ html,
 body {
   padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell,
```

Comment:
I'm guessing we need to configure prettier.

Could you copy/paste our prettier config from P3 to the package.json file here?

Reply:
Ah, nevermind, I guess I didn't run prettier on this file when I created it. My bad.